android/app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,8 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.READ_PROFILE" />
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
```

Comment:
Do we want those?

Reply:
https://rnfirebase.io/docs/v4.2.x/notifications/android#Update-Android-Manifest